mobile/src/modules/auth/store/actions.ts

```diff
@@ -81,12 +82,26 @@ export const createOfflineAccount = createActionFn<string, Account>(
     }
 
     // TODO: make fields optional in @burst package
-    // @ts-ignore
-    return {
+    return new Account({
       type: 'offline', // TODO: make type enum in @burst package
       account,
       accountRS
-    };
+    });
+  }
+);
+
+export const hydrateAccount = createActionFn<Account, Promise<Account>>(
+  async (dispatch, getState, account) => {
+
+    const api = composeApi(new ApiSettings('https://wallet1.burst-team.us:2083', 'burst'));
```

Comment:
todo: move this stuff to a config service or something

Reply:
just an idea:
0) make interface `BurstSettings` extends `ApiSettings` in src/core/interfaces, so it now needs apiVersion

then src/core/store/app/reducer.ts

1) add `api` field to `AppReduxState`, like `api: Api;`
2) in AppState function add api like `const api = composeApi(appSettings.burstSettings);`

btw you can use `getState().app.appSettings.burstSettings` here instead of providing strings

Reply:
Better use Context Api:  https://reactjs.org/docs/context.html